test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -0,0 +1,232 @@
+module UnitTest {
+  use Reflection;
+  /*
+    Assert that a boolean condition is true.  If it is false, prints
+    'assert failed' and halts the program.
```

Comment:
When we switch from halting to throwing, we'll need to update this documentation

Reply:
I have updated the documentation.